tests/appsec/test_conf.py

```diff
@@ -82,40 +95,42 @@ def setup_enabled(self):
         self.r_enabled = weblog.get("/waf/", headers={"User-Agent": "Arachni/v1"})
 
     def test_enabled(self):
-        """ test DD_APPSEC_ENABLED = true """
+        """test DD_APPSEC_ENABLED = true"""
         interfaces.library.assert_waf_attack(self.r_enabled)
 
     def setup_disabled(self):
         self.r_disabled = weblog.get("/waf/", headers={"User-Agent": "Arachni/v1"})
 
-    @irrelevant(library="ruby", weblog_variant="rack", reason="it's not possible to auto instrument with rack")
+    @irrelevant(
+        library="ruby", weblog_variant="rack", reason="it's not possible to auto instrument with rack",
+    )
     @missing_feature(
         context.weblog_variant in ["sinatra14", "sinatra20", "sinatra21", "uds-sinatra"],
         reason="Conf is done in weblog instead of library",
     )
     @scenarios.appsec_disabled
     def test_disabled(self):
-        """ test DD_APPSEC_ENABLED = false """
+        """test DD_APPSEC_ENABLED = false"""
         interfaces.library.assert_no_appsec_event(self.r_disabled)
 
     def setup_appsec_rules(self):
         self.r_appsec_rules = weblog.get("/waf", headers={"attack": "dedicated-value-for-testing-purpose"})
 
     @scenarios.appsec_custom_rules
     def test_appsec_rules(self):
-        """ test DD_APPSEC_RULES = custom rules file """
+        """test DD_APPSEC_RULES = custom rules file"""
         interfaces.library.assert_waf_attack(self.r_appsec_rules, pattern="dedicated-value-for-testing-purpose")
 
     def setup_waf_timeout(self):
-        long_payload = "?" + "&".join(f"{k}={v}" for k, v in ((f"key_{i}", f"value{i}") for i in range(1000)))
+        long_payload = "?" + "&".join(f"{k}={v}" for k, v in ((f"key_{i}", f"value{i}") for i in range(10_000)))
```

Comment:
Do we want to do this?

Reply:
Apparently, 1000 was not enough for the new python framework to create a timeout on the github actions. Not sure why this is apparently faster on that.